driver/driver.go

```diff
@@ -619,6 +612,19 @@ func (p *S3fsPlugin) mountInternal(mountRequest interfaces.FlexVolumeMountReques
 		args = append(args, "-o", "ro")
 	}
 
+	var tlsCipherSuite string
+	if len(strings.TrimSpace(options.TLSCipherSuite)) != 0 && options.TLSCipherSuite != "default" {
+		tlsCipherSuite = options.TLSCipherSuite
+	} else {
+		tlsCipherSuite = ""
+	}
+
+	// Add cipher_suite option only if the value is !=default or nonempty
+
+	if tlsCipherSuite != "" {
+		args = append(args, "-o", "cipher_suites="+tlsCipherSuite)
+	}
+
```

Comment:
Can we replace above code with below snippet?
```
var tlsCipherSuite string	if len(strings.TrimSpace(options.TLSCipherSuite)) != 0 &amp;&amp; options.TLSCipherSuite != "default" {		tlsCipherSuite = options.TLSCipherSuite	} else {		tlsCipherSuite = ""	}
	// Add cipher_suite option only if the value is !=default or nonempty
	if tlsCipherSuite != "" {		args = append(args, "-o", "cipher_suites="+tlsCipherSuite)	}
```

Reply:
Agree